ui/src/components/home/SongTab.tsx

```diff
@@ -0,0 +1,61 @@
+import { Box, Typography, TextField } from '@mui/material';
+import MusicNoteIcon from '@mui/icons-material/MusicNote';
+import SearchIcon from '@mui/icons-material/Search';
+import { FC, ReactElement } from 'react';
+
+const SongTab: FC = (): ReactElement => {
+  return (
+    <Box
+      sx={{
+        padding: '16px',
```

Comment:
just a side note, think woud be better to be more consistent either use px / em. It will get confusing in the future when we have other components too, to use px at some and em at some.